tdp/core/models/deployment_log.py

```diff
@@ -285,19 +294,35 @@ def from_stale_components(
             targets=list([operation.name for (operation, _) in operation_hosts]),
             hosts=None,
             extra_vars=None,
+            rolling_interval=rolling_interval,
             deployment_type=DeploymentTypeEnum.RECONFIGURE,
             state=DeploymentStateEnum.PLANNED,
         )
-        deployment_log.operations = [
-            OperationLog(
-                operation=operation.name,
-                operation_order=i,
-                host=host,
-                extra_vars=None,
-                state=OperationStateEnum.PLANNED,
+        operation_order = 1
+        for operation, host in operation_hosts:
```

Comment:
Can't we use enumerate here as there is no nested logic anymore?

Reply:
There is no nested logic.